components/select/nz-select.component.spec.ts

```diff
@@ -162,6 +162,16 @@ describe('nz-select component', () => {
       expect(targetElement.style.width).toBe('');
       expect(targetElement.style.minWidth).toBe('10px');
     }));
+    it('should custom template work', () => {
```

Comment:
should test in both single and multiple modes

Reply:
done